addOns/pscanrules/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 - Maintenance changes.
 - Migrate CSP Scanner into the main passive scan release package (promoting it to Release). Upgrade Salvation (dependency) to 2.6.0.
+- Application Error scanner change for HTTP 500. Alert changed to low for HTTP 500 and, avoid Alert for threshold HIGH  
```

Comment:
For the second sentence let’s go with something more like “Alert changed to low risk for HTTP 500, and not raised at all when Threahold is High.”